README.md

```diff
@@ -135,3 +135,8 @@ Here's an example:
 *Note that it requires your service to output JSON formatted logs with a
 structure that ecs-logs recognize.*
 
+### Proxy
+
+To send your logs through a proxy, you can set the `HTTP_PROXY` or `SOCKS_PROXY` environment variable.
```

Comment:
Go also supports the `HTTPS_PROXY` environment variable to tunnel https connections ;)